explorer/models/taskNode.ts

```diff
@@ -31,35 +31,35 @@ export class TaskRootNode extends NodeBase {
         }
     }
 
-    /* Making a list view of BuildTaskNodes, or the Build Tasks of the current registry */
-    public async getChildren(element: TaskRootNode): Promise<BuildTaskNode[]> {
-        const buildTaskNodes: BuildTaskNode[] = [];
-        let buildTasks: ContainerModels.BuildTask[] = [];
+    /* Making a list view of TaskNodes, or the Tasks of the current registry */
+    public async getChildren(element: TaskRootNode): Promise<TaskNode[]> {
+        const taskNodes: TaskNode[] = [];
+        let tasks: ContainerModels.Task[] = [];
         const client = AzureUtilityManager.getInstance().getContainerRegistryManagementClient(element.subscription);
         const resourceGroup: string = acrTools.getResourceGroupName(element.registry);
-        buildTasks = await client.buildTasks.list(resourceGroup, element.registry.name);
-        if (buildTasks.length === 0) {
-            vscode.window.showInformationMessage(`You do not have any Build Tasks in the registry, '${element.registry.name}'. You can create one with ACR Build. `, "Learn More").then(val => {
+        tasks = await client.tasks.list(resourceGroup, element.registry.name);
+        if (tasks.length === 0) {
+            vscode.window.showInformationMessage(`You do not have any Tasks in the registry, '${element.registry.name}'. You can create one with ACR Build. `, "Learn More").then(val => {
                 if (val === "Learn More") {
                     opn('https://aka.ms/acr/buildtask');
```

Comment:
>https://aka.ms/acr/buildtask [](start = 25, length = 28)

I create a new short link you can change to '[http://aka.ms/acr/task](http://aka.ms/acr/task)'. It's pointing to the old doc now I will redirect to to the task document once it's online.